AuthenticationAndAuthorization/DataAccounts/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AutoMapper;
+using DataAccounts.Repositories;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.Win32;
+using System.Data;
+
+namespace DataAccounts
+{
+    public class UserRepository : IUserRepository
+    {
+        private readonly UserManager<User> _userManager;
+        private readonly IMapper _mapper;
```

Comment:
Mapping is BusinessLogic concern. Let's move mapping to Services layer and pass to repositories Entities